src/core/CCList.mli

```diff
@@ -576,73 +595,84 @@ val range : int -> int -> int t
     both for decreasing and increasing ranges. *)
 
 val range' : int -> int -> int t
-(** Like {!range} but the second bound is excluded.
+(** [range' i j] is like {!range} but the second bound [j] is excluded.
     For instance [range' 0 5 = [0;1;2;3;4]]. *)
 
 val (--) : int -> int -> int t
-(** Infix alias for [range]. *)
+(** [i -- j] iterates on integers from [i] to [j] included.
```

Comment:
`(--)` does not really iterate, it creates a list.

Reply:
Modified.